apps/nextra/pages/en/build/create-aptos-dapp/templates/digital-asset.mdx

```diff
@@ -98,77 +129,114 @@ Metadata files are `json` files that hold the Collection data and each of the NF
       <FileTree.File name="2.jpeg" />
       <FileTree.File name="3.jpeg" />
     </FileTree.Folder>
-    <FileTree.Folder name="metadatas">
+    <FileTree.Folder name="metadata">
       <FileTree.File name="1.json" />
       <FileTree.File name="2.json" />
       <FileTree.File name="3.json" />
     </FileTree.Folder>
   </FileTree.Folder>
 </FileTree>
+</Steps>
 
-<Callout type="info">
-  Make sure all image files have the same file extension. If some images will
-  have `jpeg` extension and some `jpg` extension, the upload will fail.
-</Callout>
+## Publish the Move Contract
 
-{ /* TODO: change ERC-1155 once finish rewriting the NFT standards page */ }
+<Steps>
+### Prepare an account, or create one with an Aptos wallet like [Petra](https://petra.app/docs/use#create-a-new-account).
+You will need this to run the contract to create a collection. Make sure to make the account on the same network as your template project.
+
+### Use the below command to create the contract to mint your collection.
 
-We recommend the developers follow the [ERC-1155 off-chain data](https://eips.ethereum.org/EIPS/eip-1155) schema to format their JSON files. One thing to note, since the image field will be updated with the Arweave link after this step, feel free to leave it empty.
+```bash npm2yarn
+npm run move:init
+```
 
-Example of `collection.json` file
+The CLI will prompt you to provide the secret for your wallet in order for the tool to make the collection on your behalf.
```

Comment:
```suggestion
The CLI will prompt you to provide the private key for your wallet in order for the tool to make the collection on your behalf.  If you provide no input, it will generate one for you.
```